script.js

```diff
@@ -1,3 +1,114 @@
+  technologies: ['codekit', 'GitHub', 'JavaScript', 'Bootstrap', 'Codepen'],
+  live: 'https://vgonma.github.io/personalportfolio/',
+  github: 'https://github.com/Vgonma',
+};
+
+const P3 = {
+  name: 'Project 3',
+  description:
+    'Lorem ipsum dolor sit amet consectetur adipisicing elit. Debitis aliquid cupiditate vitae tempore temporibus quo, eum pariatur. Neque sed ab in et architecto? In cumque obcaecati temporibus incidunt. A assumenda explicabo perspiciatis soluta tempora facilis suscipit culpa quo, alias nesciunt omnis similique quisquam fuga consectetur error laboriosam! Quis, quos repellendus.',
+  image: ['./images/pop-up-image.svg', './images/snapshot-portfolio.svg'],
+  technologies: ['codekit', 'GitHub', 'JavaScript', 'Bootstrap', 'Codepen'],
+  live: 'https://vgonma.github.io/personalportfolio/',
+  github: 'https://github.com/Vgonma',
+};
+
+const P4 = {
+  name: 'Project 4',
+  description:
+    'Lorem ipsum dolor sit amet consectetur adipisicing elit. Debitis aliquid cupiditate vitae tempore temporibus quo, eum pariatur. Neque sed ab in et architecto? In cumque obcaecati temporibus incidunt. A assumenda explicabo perspiciatis soluta tempora facilis suscipit culpa quo, alias nesciunt omnis similique quisquam fuga consectetur error laboriosam! Quis, quos repellendus.',
+  image: ['./images/pop-up-image.svg', './images/snapshot-portfolio.svg'],
+  technologies: ['codekit', 'GitHub', 'JavaScript', 'Bootstrap', 'Codepen'],
+  live: 'https://vgonma.github.io/personalportfolio/',
+  github: 'https://github.com/Vgonma',
+};
+
+const P5 = {
+  name: 'Project 5',
+  description:
+    'Lorem ipsum dolor sit amet consectetur adipisicing elit. Debitis aliquid cupiditate vitae tempore temporibus quo, eum pariatur. Neque sed ab in et architecto? In cumque obcaecati temporibus incidunt. A assumenda explicabo perspiciatis soluta tempora facilis suscipit culpa quo, alias nesciunt omnis similique quisquam fuga consectetur error laboriosam! Quis, quos repellendus.',
+  image: ['./images/pop-up-image.svg', './images/snapshot-portfolio.svg'],
+  technologies: ['codekit', 'GitHub', 'JavaScript', 'Bootstrap', 'Codepen'],
+  live: 'https://vgonma.github.io/personalportfolio/',
+  github: 'https://github.com/Vgonma',
+};
+
+const P6 = {
+  name: 'Project 6',
+  description:
+    'Lorem ipsum dolor sit amet consectetur adipisicing elit. Debitis aliquid cupiditate vitae tempore temporibus quo, eum pariatur. Neque sed ab in et architecto? In cumque obcaecati temporibus incidunt. A assumenda explicabo perspiciatis soluta tempora facilis suscipit culpa quo, alias nesciunt omnis similique quisquam fuga consectetur error laboriosam! Quis, quos repellendus.',
+  image: ['./images/pop-up-image.svg', './images/snapshot-portfolio.svg'],
+  technologies: ['codekit', 'GitHub', 'JavaScript', 'Bootstrap', 'Codepen'],
+  live: 'https://vgonma.github.io/personalportfolio/',
+  github: 'https://github.com/Vgonma',
+};
+
+const projects = [P1, P2, P3, P4, P5, P6];
```

Comment:
- [ optional ] You did well and its working but I think creating a constant for every object 
1 - is taking you too much time
2 - is adding execution time
3 - is making your code more complex thereby giving room for bugs
I think it would have been better to just create the `projects`  constant as an array of objects i.e
```
const projects = [
  {
    ...
  },
  {
    ...
  },
  {
    ...
  }
...
]
```